shared/src/ledger/storage/write_log.rs

```diff
@@ -334,6 +334,7 @@ impl WriteLog {
             HashMap::with_capacity(100),
         );
         self.block_write_log.extend(tx_write_log);
+        self.take_ibc_event();
```

Comment:
ahhh, good catch!

Reply:
I had to pull Yuji in a call to hunt that down. Luckily missing it fails a test.